addon/mixins/sortable-item.js

```diff
@@ -377,17 +384,16 @@ export default Mixin.create({
     @return Promise
   */
   _waitForTransition() {
-    return new Promise(resolve => {
-      run.next(() => {
-        let duration = 0;
-
-        if (this.get('isAnimated')) {
-          duration = this.get('transitionDuration');
-        }
-
-        run.later(this, resolve, duration);
+    if (this.get('isAnimated')) {
```

Comment:
Maybe I’m reading this wrong, but it looks like the condition is the wrong way around? Shouldn’t it resolve immediately when it *isn’t* animated?